app/databases/show/operations/route.js

```diff
@@ -1,14 +1,19 @@
 import Ember from 'ember';
+import Paginated from "diesel/mixins/routes/paginated";
 
-export default Ember.Route.extend({
-  model: function(){
+export default Ember.Route.extend(Paginated, {
+  model: function(params, transition){
     var db = this.modelFor('databases.show');
+    var page = parseInt(transition.queryParams.page || 1, 10);
```

Comment:
that's correct — the queryParam is coerced into params as its property name. In this case `params.currentPage`. I'll push a commit in a second.